.spelling

```diff
@@ -238,15 +239,6 @@ CVE-2019-9513
 CVE-2019-9514
 CVE-2019-9515
 CVE-2019-9518
-CVE-2019-15605
```

Comment:
I just used `sort` on this list

Reply:
could you do a numeric sort, as there's no leading 0?